src/scss/webmapviewer-bootstrap-theme.scss

```diff
@@ -2,6 +2,13 @@ $red: #dc3545 !default;
 $primary: $red;
```

Comment:
could we straight use the newly defined `$venetian-red` here?

Reply:
I updated the custom `$red` to `$venetian-red`. I kept the styles that were using `$red`.